pkg/install/install.go

```diff
@@ -20,6 +20,51 @@ import (
 )
 
 var servingVersion = "0.23.0"
+var kourierVersion = "0.23.0"
```

Comment:
Don't know how I fell hard coding the version, I would feel better if we open an issue as tech debt to address this.

Ideally we should dynamically query that's the version of the latest release.